lib/checks/color/color-contrast.json

```diff
@@ -21,7 +21,11 @@
     "impact": "serious",
     "messages": {
       "pass": "Element has sufficient color contrast of ${data.contrastRatio}",
-      "fail": "Element has insufficient color contrast of ${data.contrastRatio} (foreground color: ${data.fgColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Expected contrast ratio of ${data.expectedContrastRatio}",
+      "fail": {
+        "fgOnBgColor": "Element has insufficient color contrast of ${data.contrastRatio} (foreground color: ${data.fgColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Expected contrast ratio of ${data.expectedContrastRatio}",
+        "fgOnShadowColor": "Element has insufficient color contrast of ${data.contrastRatio} (foreground color: ${data.fgColor}, text-shadow color: ${data.shadowColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Expected contrast ratio of ${data.expectedContrastRatio}",
```

Comment:
These are very subtle differences in the text, we might want to improve the message itself (not just the numbers we display) to tell the user that shadow text is being applied rather than having the user glean that from the text inside the parenthesis.